mathpple.map is nowhere activated, upstream put it into ams.tpm
---
 texmf-dist/tpm/ams.tpm |    1 +
 1 file changed, 1 insertion(+)

Index: texlive-base-2007/texmf-dist/tpm/ams.tpm
===================================================================
--- texlive-base-2007.orig/texmf-dist/tpm/ams.tpm	2007-06-05 10:19:58.000000000 +0200
+++ texlive-base-2007/texmf-dist/tpm/ams.tpm	2007-06-05 10:20:09.000000000 +0200
@@ -303,6 +303,7 @@
 texmf-dist/tpm/ams.tpm
     </TPM:RunFiles>
     <TPM:Installation>
+      <TPM:Execute function="addMap" parameter="mathpple.map"/>
       <TPM:Execute function="addMap" mode="mixed" parameter="ams-bsr-interpolated.map"/>
       <TPM:Execute function="addMap" mode="mixed" parameter="ams-bsr.map"/>
       <TPM:Execute function="addMap" mode="mixed" parameter="ams-cmcsc-bsr-interpolated.map"/>
